src/index.ts

```diff
@@ -16,10 +16,16 @@ type Experience = {
     isControl: boolean,
     id: number,
     callback: string,
-    variation: number
+    variation: number,
     shown: () => void
 }
 
+type Placement = {
+    content: object,
```

Comment:
Is `content` a property of returned Placement?

In other places (for example below) , Placement is documented by example:
```
{
     *   "image": "https://image.store.com/images/example.jpeg",
     *   "message": "Hello World",
     *   "url": "https://www.qubit.com"
     *   "impressionUrl": "https://api.qubit.com/placements/callback?data=ggW4eyJtZXRhIjp7ImlkIjo",
     *   "clickthroughUrl": "https://api.qubit.com/placements/callback?data=mQW4eyJtZXRhIjp7Imlkx"
     * }
```

Reply:
Somehow mapping between received JSON to this `Placement` structure has worked with this `content` property. However once I've changed it to explicit parameters definition it works as well.

Reply:
I guess, mapping was working, because example application is written in pure Javascript, not in Typescript, so type declarations are completely ignored.

Reply:
Isn't "index.ts" filename indicating that this is actually typescript?

Reply:
Yes. This is Typescript. API of SDK is written in typescript, but example application App.js is written in Javascript. That is why, there is no compile-time checking of this type at the moment. You can write anything you want in Placement type, it will not cause type-checking errors.